psql2mysql/__init__.py

```diff
@@ -167,13 +173,18 @@ def disable_constraints(self):
             "SET SESSION foreign_key_checks='OFF'"
         )
 
+    @profile
```

Comment:
I'd prefer if this was able to be turned off.

Reply:
I might remove the memory profiling. It was there while I was comparing, it's easy to add. So shouldn't be there by default. I'll remove it in the next version.